Adyen/Helpers/UIViewControllerHelpers.swift

```diff
@@ -39,12 +24,13 @@ public extension AdyenScope where Base: UIViewController {
 
     /// :nodoc:
     func present(_ viewController: UIViewController, animated: Bool, completion: (() -> Void)? = nil) {
+        let topPresenter = self.topPresenter
         if topPresenter.isBeingPresented || topPresenter.isBeingDismissed {
-            DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(10)) {
-                self.present(viewController, animated: animated, completion: completion)
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
10 milisec could be not enough. This is used for our Demo only, so should be fine